lametro/models.py

```diff
@@ -191,6 +191,12 @@ def upcoming_board_meeting(cls):
 
     @classmethod
     def current_meeting(cls):
+        
+        #use this line to hardcode one current event
+        #return Event.objects.filter(start_time='2017-06-15 13:30:00-05')
+        #use this line to hardcode two current events
+        #return Event.objects.filter(start_time='2017-11-30 11:00:00-06')
```

Comment:
I'd actually like to see these comments consolidated with the "docstring" (lines 201 ff.)....something, like, "To hardcode current event(s) for testing, then return these querysets...." (etc etc etc).